build.gradle

```diff
@@ -10,39 +12,38 @@ version '1.0'
 mainClassName = 'io.beekeeper.xbot.Main'
 applicationDefaultJvmArgs = ["-Xms64M", "-Xmx128M"]
 
-sourceCompatibility = 1.8
+sourceCompatibility = 1.11
 
 repositories {
     mavenCentral()
     maven { url 'https://s3-eu-west-1.amazonaws.com/beekeeper-marketplace-sdk/maven2/release/' }
 }
 
 dependencies {
-    compile 'io.beekeeper:beekeeper-sdk:2.4.0'
-    compile 'io.beekeeper:beekeeper-chatbot-sdk:2.4.0'
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    implementation 'io.beekeeper:beekeeper-sdk:3.7.1'
```

Comment:
Just adding plugin is not enough in my opinion:

![image](https://user-images.githubusercontent.com/35264502/146335481-b835c8ec-d90f-4425-8df8-05407d3788da.png)

Reply:
Just a test comment

Reply:
...